actnow/profiles/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+from .validators import phone_regex
+
+
+class UserProfile(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    bio = models.TextField(
+        verbose_name="About me", max_length=255, blank=True, null=True
+    )
+    location = models.TextField()
+    phone_number = models.CharField(validators=[phone_regex], max_length=17, blank=True)
+    socials = models.URLField(blank=True, null=True)
```

Comment:
```suggestion
    social_media_link = models.URLField(blank=True, null=True)
```

socials sounds weird but you can keep it if it sounds alright to you.

The most important thing is we need to validate the URL that it's actually a social media URL (checking domain should be enough).

Reply:
I'm not sure about this. Is there a number of social media accounts we should work with? E.g Facebook, Twitter, e.t.c

Reply:
Only the usual suspects @esirK ... FB, Twitter, WhatsApp, etc. Lets limit it to the ones we support sharing content to.